cclib/parser/turbomoleparser.py

```diff
@@ -114,7 +222,7 @@ def extract(self, inputfile, line):
             homos=int(temp[1])
 
         if line[0:6] == "$basis":
-            print "Found basis"
+            print("Found basis")

```

Comment:
Is this left over debugging code?

Reply:
Had to make this change, since the parser kept failing tests for python 3.